src/component/card/card.scss

```diff
@@ -1,10 +1,32 @@
 @import '@/style/theme.scss';
 
+$backgrounds: (
+  'primary-color': 'knowledge-tree',
+  'primary-color-variant-1': 'data-services-tree',
+  'primary-color-variant-2': 'data-spaces-tree',
+  'primary-color-variant-3': 'applications-tree'
+);
+
 .okp4-dataverse-portal-card-main {
   @include with-theme() {
-    background-color: themed('card');
     border-radius: 20px;
     display: flex;
     align-items: center;
+
+    @each $background-key, $background-value in $backgrounds {
+      &.#{$background-key} {
+        background: themed($background-key)
+          url('@/asset/images/#{$background-value}.webp')
+          no-repeat;
+
+        &:hover {
```

Comment:
I will have added a `cursor: pointer;` to highlight the idea that these cards are CTA

Reply:
@AnjaMis 
Does this change has been taken into account?

Reply:
it's ok I resolve the comment